docs/api-reference.md

```diff
@@ -201,7 +201,7 @@ _Appears in:_
 
 _Underlying type:_ `string`
 
-
+KongProtocol is a valid Kong protocol This alias is necessary to deal with https://github.com/kubernetes-sigs/controller-tools/issues/342
```

Comment:
@czeslavo This is correct, isn't it? Or was it meant to be hidden?

Reply:
I think so, don't understand why it was ignored before 🤔